test/LowerColorado_TX/test_AnA.yaml

```diff
@@ -9,13 +9,14 @@ network_topology_parameters:
     #----------
     supernetwork_parameters:
         #----------
-        geo_file_path: domain/RouteLink.nc
+        geo_file_path: /glade/scratch/adamw/CONUS_1yr/domain/RouteLink_NWMv2.1.nc 

```

Comment:
again, these paths are being modified from a self contained test that exists in the repository, we should change these paths if we expect a user/developer to pull this repository and run the included LowerColorado_TX test